crates/compiler/mono/src/drop_specialization.rs

```diff
@@ -1552,7 +1543,7 @@ where
         }
     }
 
-    fn contains(&self, symbol: &K) -> bool {
+    fn contained(&self, symbol: &K) -> bool {
```

Comment:
why change this name?

Reply:
Because it keeps keys around with a count of 0. So the map *contained* the value before. But might not now.

Reply:
hmm then maybe a doc comment on the `CountingMap` explaining its purpose would be nice

Reply:
because by the looks of it, the function returns true if the key is in the map at all. No check for the value being 0 is performed